packages/react/src/hooks/__tests__/useAuth.test.ts

```diff
@@ -78,7 +78,7 @@ describe('useAuth', () => {
 
       const { result, waitForNextUpdate } = renderHook(() => useAuth());
 
-      await waitForNextUpdate();
+      await act(async () => await waitForNextUpdate());
```

Comment:
Think `waitForNextUpdate` returns `Promise<void>`, does the callback need to return?

Reply:
In some cases yes but took out the async code where it wasn't necessary

Reply:
Think my question may have been lost in translation, an arrow function with curly brackets creates an implicit return:
```
const returnThing = () => 'thing';

console.log(returnThing());
// 'thing';
```

In the case of:

```
await act(async () => await waitForNextUpdate());
```

`act` do anything with the return of its callback, so the implicit return is extraneous